src/main/java/org/apache/commons/compress/compressors/lzma/LZMAUtils.java

```diff
@@ -146,7 +146,7 @@ public static String getCompressedFilename(final String fileName) {
      * <p>This defaults to {@code false} in an OSGi environment and {@code true} otherwise.</p>
      * @param doCache whether to cache the result
      */
-    public static void setCacheLZMAAvailablity(final boolean doCache) {
```

Comment:
Even if its spelling is wrong, we cannot change the name of a public method as it would break backwards compatibility.

Reply:
> Even if its spelling is wrong, we cannot change the name of a public method as it would break backwards compatibility.

done.